tests/dragonfly/replication_test.py

```diff
@@ -1208,9 +1208,17 @@ async def seed():
 
     # Give the seeder a bit of time.
     await asyncio.sleep(1)
-    await c_replica.execute_command(f"REPLTAKEOVER 5")
```

Comment:
In this test we want to check replicatakeover while the seeder is running i.e commands are executed, so please do not do this change.